### DISCUSSION TITLE
Dual Extruder support for Ender 3 v2 and Jyers?

### DISCUSSION THREAD

**Discussion by Redleg1096** [Jyers/Marlin#1342] [Ideas]
Is there a version of Jyers that supports dual extruders with a mixing hot end by chance?
I have a BTT SKR E3 Turbo hanging out on standby for my Ender 3 v2 just waiting for such a firmware update.

**Comment by Cal-LF** [comment]
Dual extruder feature will have to be a compiled firmware of your own.
